### ISSUE TITLE
Send MDM Commands for computer is not supported in this library?

### ISSUE THREAD

**Issue by Neil-Liang** [jssimporter/python-jss#33]
Would like to use it for device lock :)

**Comment by sheagcraig** [comment]
Hey @Neil-Liang 
If it's supported in the API, it can be supported here. I've never tried to do that, but I think it should be possible. Let me look into that.

**Comment by sheagcraig** [comment]
@Neil-Liang Having this would definitely be super-useful.

Basically, what it would take to implement would be examples of the XML for each of the supported computer command (and mobile device command probably while we're at it).

python-jss would add a `new` method for each probably, which creates the appropriate XML in preparation to POST.

Unfortunately, I'm switching to a non-casper environment in the very near-future, so my ability to test this is basically gone. If you can get me the XML I can either implement it in the testing version and have you test, or get another collaborator to work on it.

**Comment by sheagcraig** [comment]
@Neil-Liang did you ever come up with the XML needed to send the MDM commands?

**Comment by mosen** [comment]
There are 3 variations on the request, and a total of 4 commands supported.

**BlankPush**
POST `/JSSResource/computercommands/command/BlankPush/id/148,149`
no body required
Issues BlankPush for computer IDs 148 and 149.

**UnmanageDevice**
POST `/JSSResource/computercommands/command/UnmanageDevice/id/148`
no body required

**DeviceLock**
POST `/JSSResource/computercommands/command/DeviceLock/passcode/<6 digit pin>/id/148`
No body required

**EraseDevice**
POST `/JSSResource/computercommands/command/EraseDevice/passcode/<6 digit pin>/id/148`
No body required

**BULK UPDATE**
POST `/JSSResource/computercommands/command/<command>`
Unsure about body format. tested several XML formats and no success yet.
This endpoint required if you want to do bulk commands.

**Comment by mosen** [comment]
Bulk Update Body

```
<computer_command>
	<general>
		<command>BlankPush</command>
		<passcode>123456 (only required for DeviceLock and EraseDevice)</passcode>
	</general>
	<computers>
		<computer>
			<id>148</id>
		</computer>
	</computers>
</computer_command>
```

**Comment by mosen** [comment]
Fixed in PR#78
